xinference/deploy/worker.py

```diff
@@ -22,10 +22,24 @@
 from xoscar import MainActorPoolType
 
 from ..core.worker import WorkerActor
+from ..deploy.utils import rollover_log_file
 
 logger = logging.getLogger(__name__)
 
 
+def _config_logging(address: str, supervisor_address: str):
+    worker_host = address.split(":")[0]
+    supervisor_host = supervisor_address.split(":")[0]
+    # Local deployment, worker not do rollover
+    if (
+        worker_host == "127.0.0.1"
+        or worker_host == "0.0.0.0"
+        or worker_host == supervisor_host
+    ):
+        return
+    rollover_log_file(logger, "Xinference worker ")
```

Comment:
A more effective approach might be to direct the supervisor and worker logs to separate files. What are your thoughts on this?

Reply:
> A more effective approach might be to direct the supervisor and worker logs to separate files. What are your thoughts on this?

This approach requires that the filename to be written to the log be passed in dynamically, rather than determined at the outset, and may require relatively larger changes to existing code. I'm not sure it needs to be done in the current PR.